product/domain/productBasics.go

```diff
@@ -85,11 +85,12 @@ type (
 
 	// Saleable are properties required for being selled
 	Saleable struct {
-		IsSaleable      bool
-		SaleableFrom    time.Time
-		SaleableTo      time.Time
-		ActivePrice     PriceInfo
-		AvailablePrices []PriceInfo
+		IsSaleable         bool
+		SaleableFrom       time.Time
+		SaleableTo         time.Time
+		ActivePrice        PriceInfo
+		AvailablePrices    []PriceInfo
+		ActiveLoyaltyPrice LoyaltyPriceInfo
```

Comment:
What's the reason behind touching the domain model? thought we would only expose the already existing `LoyaltyPrices []LoyaltyPriceInfo`slice via graphql.